ui/StatusQ/src/StatusQ/Components/StatusMessageHeader.qml

```diff
@@ -15,9 +15,7 @@ Item {
     property StatusMessageSenderDetails sender: StatusMessageSenderDetails { }
 
     property alias displayNameLabel: primaryDisplayName
-    property alias secondaryNameLabel: secondaryDisplayName
-    property alias tertiaryDetailsLabel: tertiaryDetailText
-    property alias timestamp: timestampText
+    property double timestamp: Date.now()
```

Comment:
0 is just fine as default

Reply:
Sorry, my bad. 0 is not fine. IDK why I had the impression that Date(0) == Date(). Basically Date.now() is the default for this property, but if it's overridden from above Date.now() won't be called. I would leave the default Date.now() because it makes more sense as opposed to 1970. The same for all Date.now()

Reply:
this property should always be overwritten no? So if we ever show 1970, it means we didn't something wrong no?

Reply:
Yes, if we add 0 as default showing 1970 points out an issue and it would be easier to spot it.

Reply:
Done. All dates are defaulted to 0.